app/client/cli/servicer.go

```diff
@@ -0,0 +1,284 @@
+	}
+
+	req := rpc.SessionRequest{
+		AppAddress: appAddress,
+		Chain:      chain,
+		// TODO: Geozone
+		SessionHeight: currentHeight,
+	}
+
+	client, err := rpc.NewClientWithResponses(remoteCLIURL)
+	if err != nil {
+		return nil, fmt.Errorf("Error getting current session for app/chain/height: %s/%s/%d: %w", appAddress, chain, currentHeight, err)
+	}
+
+	resp, err := client.PostV1ClientGetSessionWithResponse(ctx, req)
+	if err != nil {
+		return nil, fmt.Errorf("Error getting current session with request %v: %w", req, err)
+	}
+	// TODO: refactor boiler-plate code
+	if resp.HTTPResponse.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("Error getting current session: Unexpected status code %d for request %v", resp.HTTPResponse.StatusCode, req)
+	}
+
+	if resp.JSON200 == nil {
+		return nil, fmt.Errorf("Error getting current session: Unexpected response %v", resp)
+	}
+
+	return resp.JSON200, nil
+}
+
+// TODO: reuse this function in the query commands
+func getCurrentHeight(ctx context.Context) (int64, error) {
+	client, err := rpc.NewClientWithResponses(remoteCLIURL)
+	if err != nil {
+		return 0, fmt.Errorf("Error getting current height: %w", err)
+	}
+	resp, err := client.GetV1QueryHeightWithResponse(ctx)
+	if err != nil {
+		return 0, fmt.Errorf("Error getting current height: %w", err)
+	}
+	if resp.HTTPResponse.StatusCode != http.StatusOK {
+		return 0, fmt.Errorf("Error getting current height: Unexpected status code %d", resp.HTTPResponse.StatusCode)
+	}
+
+	if resp.JSON200 == nil {
+		return 0, fmt.Errorf("Error getting current height: Unexpected response %v", resp)
+	}
+
+	return resp.JSON200.Height, nil
+}
+
+// TODO: (localnet) Publish Servicer(s) Host and Port as env. vars in K8s: similar to Validators
+// TODO: (REFACTOR) should we move package-level variables (e.g. remoteCLIURL) to a cli object?
+func sendTrustlessRelay(ctx context.Context, servicerUrl string, relay rpc.RelayRequest) (*rpc.PostV1ClientRelayResponse, error) {
+	client, err := rpc.NewClientWithResponses(servicerUrl)
+	if err != nil {
+		return nil, err
+	}
+
+	return client.PostV1ClientRelayWithResponse(ctx, relay)
+}
+
+// TODO: (NICE) allow reading the relay request from the command line arguments AND from a file
+func buildRelay(payload string, appPrivateKey crypto.PrivateKey, session rpc.Session, servicer rpc.ProtocolActor) (rpc.RelayRequest, error) {
+	// TECHDEBT: This is mostly COPIED from pocket-go: we should refactor pocket-go code and import this functionality from there instead.
+	relayPayload := rpc.Payload{
+		Data:   payload,
+		Method: "POST",
+		// TODO: Path: load Path field from the corresponding Blockchain (e.g. database)
+		// TODO: set Headers
+	}
+
+	relayMeta := rpc.RelayRequestMeta{
+		BlockHeight: session.SessionHeight,
+		// TODO: use Identifiable for Chain in Session (or string for Chain in Relay Meta)
+		Chain: rpc.Identifiable{
+			Id: session.Chain,
+		},
+		ServicerPubKey: servicer.PublicKey,
+		// TODO: Geozone
```

Comment:
Add the same github issue related to geozones as what I pasted to above

Reply:
Thanks. Fixed.